src/types.mligo

```diff
@@ -222,13 +174,15 @@ type transfer_ownership_param = address
 type custom_ep_param = (string * bytes)
 
 type propose_params =
-  { frozen_token : nat
+  [@layout:comb]
+  { from_ : address
```

Comment:
@rinn7e I didn't get why we need to add the `from_` field here and for `vote_param`, can you please clarify?

Reply:
This is related to the new role `delegate` which is similar to FA2 `operator` but instead can call `propose` and `vote` entrypoints. Since we can't use `sender` anymore, we need a new field. More discussion here: https://serokell.slack.com/archives/C019PCGKTK2/p1622222486020100?thread_ts=1621943302.012700&cid=C019PCGKTK2